### PR TITLE
Update supported-directives.md

### PR DESCRIPTION
**Pull request by nibinqtl**
Fixed wrong information about dropping query string to origin.

### CODE REVIEW

File: en/docs/edge-logic/supported-directives.md [hunk 1]
```diff
@@ -530,17 +530,15 @@ This is a wrapper of the [proxy_limit_rate](http://nginx.org/en/docs/http/ngx_ht
...
 ```nginx
-# when URI is not specified, URL-encoded $uri and query string will be appended by default
-origin_pass my_origin;
-eval_func $uri_uenc URL_ENCODE $uri;
-origin_pass my_origin$uri_uenc$is_args$args; #same as above
-origin_pass my_origin$uri_uenc; #to drop the query string
-origin_pass my_origin/abc$uri_uenc;
+# when URI is not specified, 
+# origin_pass my_origin; # URI-escaped $uri and the query string will be appended by default
+rewrite ^(.*) $1? break; # the question mark at the end prevents query strings being appended
```
Comment (burtonquantil) [inline en/docs/edge-logic/supported-directives.md:537]: I would add the word "from" before "being appended".

**Review by burtonquantil — APPROVED** [review]
OK to merge. I made one wording suggestion.

File: en/docs/edge-logic/supported-directives.md (hunk 1 above)
Comment (nibinqtl) [reply en/docs/edge-logic/supported-directives.md:537]: You can update the branch directly to save another round of review.

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Pull request merged**